.travis.yml

```diff
@@ -5,11 +5,15 @@ php:
 before_script:
 - sudo apt-get update -qq
 - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5 php5-curl php5-mysql php5-intl php5-gd > /dev/null
-- sudo chmod 777 -R /home/travis/build/redCOMPONENT-COM/redCORE/
+- sudo /etc/init.d/apache2 stop
+- sudo sed -i "s,www-data,$USER,g" /etc/apache2/envvars
+- sudo chown $USER:$USER -R /home/travis/build/redCOMPONENT-COM/redCORE/
```

Comment:
In the future we can switch this line to use $TRAVIS_REPO_SLUG env var (http://docs.travis-ci.com/user/ci-environment/#Environment-variables) somehow

that would allow us to copy paste this .travis.yml to other redCOMPONENT projects